tools/jolie2java/src/joliex/java/impl/JavaDocumentCreator.java

```diff
@@ -478,22 +465,24 @@ private void variableCreate(StringBuilder stringBuilder, TypeDefinition type) {
                     }
 
 
+                } else if (subType instanceof TypeChoiceDefinition){
+                    if (subType.cardinality().max() > 1) {
+                        stringBuilder.append("private List<Object> _" + subType.id() + ";\n");
+                    } else {
+                        stringBuilder.append("private Object " + subType.id() + ";\n");
```

Comment:
I think we should at least log a warning here, saying that type choices are still unsupported.